lib/modules/versioning/distro.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { DistroInfo } from './distro';
+
+describe('modules/versioning/distro', () => {
+  const di = new DistroInfo('data/ubuntu-distro-info.json');
+
+  test.each`
```

Comment:
```suggestion
  it.each`
```
please use `it` like all other tests.